Source/DayViewState.swift

```diff
@@ -17,6 +18,7 @@ public final class DayViewState {
 
   public func move(to date: Date) {
     let date = date.dateOnly(calendar: calendar)
+//    if (date == selectedDate) { return } // TODO: If necessary? Just trying to make sure that willMoveTo not be called if we are already on this page
```

Comment:
I think, adding ` if (date == selectedDate) { return }` condition is not needed and would only lead to more complicated debugging. If there are some issues in the propagating the date to the modules (e.g. some methods are called twice), it should be visible as early as possible.